### DISCUSSION TITLE
[Projects Beta] Streamline adding issues to Projects

### DISCUSSION THREAD

**Discussion by jesseko** [community/community#11148] [Projects and Issues]
1) Adding issues in the new beta is slower than on older style project boards.  Please give access to similar "Add cards" interface where you can filter the view and then easily add by dragging cards, etc.
<img width="351" alt="image" src="https://user-images.githubusercontent.com/153062/152887539-a3c065ae-1144-40ac-8680-d728430dfd91.png">

-------
2) A bulk add operation from main Issues view would be really nice.  Show a Projects menu in the actions when tickets are selected so you can check as many as you want and add them together, same as how you can do other operations on them currently. (this doesn't seem to exist for beta projects or older ones)
<img width="823" alt="image" src="https://user-images.githubusercontent.com/153062/152887458-76cb7179-de18-4cf0-9759-af6786d69a3a.png">

-------
Side note: Also can't filter Issues by which Beta projects the belong to, but can for older ones.

**Comment by jwindmuller** [comment]
The second one is similar to mine: https://github.com/github/feedback/discussions/6765
